predicators/spot_utils/spot_utils.py

```diff
@@ -620,8 +752,19 @@ def _scan_for_objects(
             self.navigate_to(waypoint_id, offset)
             for _ in range(8):
                 objects_in_view: Dict[str, Tuple[float, float, float]] = {}
-                objects_in_view_by_camera = self.get_objects_in_view_by_camera(
-                )
+                # We want to get objects in view both using AprilTags and
```

Comment:
** so we want to now always be using the vision models by default, even when the SAM grasp flag is off? is that intentional?

Reply:
nope - good catch